### DISCUSSION TITLE
Is there a way to use the colors specified in config.fish?

### DISCUSSION THREAD

**Discussion by Cokile** [PatrickF1/fzf.fish#170] [Q&A] [answered]
fzf config in config.fish
``` fish
set -x FZF_DEFAULT_OPTS '
--cycle 
--border
--height=90%
--layout=reverse 
--preview-window=wrap 
--marker=">"
--color=fg:#bdae93,bg:#32302f,hl:#83a598
--color=fg+:#ebdbb2,bg+:#3c3836,hl+:#83a598
--color=info:#fabd2f,prompt:#fabd2f,pointer:#8ec07c
--color=marker:#8ec07c,spinner:#8ec07c,header:#83a598
'
```

Using fzf directly:
![image](https://user-images.githubusercontent.com/8230448/120911362-93737980-c6b9-11eb-8b8b-c7a93a7a54a7.png)

Using fzf.fish:
![image](https://user-images.githubusercontent.com/8230448/120911378-ba31b000-c6b9-11eb-9f47-1e37eb630529.png)

As you can see, fzf.fish will use different colors. Is there a way to use the colors specified in config.fish? Or am I using fzf.fish in the wrong way?

Anyway, thanks for the awesome work. This tool really helps a lot.

**Comment by PatrickF1** [comment, marked as answer]
Hi @Cokile! The color in the second screenshot is coming from fd. If you look at https://github.com/PatrickF1/fzf.fish/blob/main/functions/__fzf_search_current_dir.fish#L6, you'll see that fd is colorizing the results before piping them to fzf. You can try setting `fzf_fd_opts` to do `--color=never` but I think it might not work because fd doesn't allow duplicate options.

Unfortunately, I can't just remove `--color=always` because it's a very helpful option for most people and I don't want people to miss out on it.

**Comment by Cokile** [comment]
Hi @PatrickF1! Thanks for the answer. The colors are helpful for me too. The problem is that the default colors have higher contrast, which makes it hard to read for me.

I've explored `fd` for a little while and it do can customize colors by set `LS_COLORS`. My current solution is using `dircolors`(which is part of `coreutils`) to set `LS_COLORS`:

``` fish
eval (gdircolors -c ~/path/to/my/dircolors)
```

Pros:
1. No need to remove the useful `--color=always`
2. Unified output when using `fzf.fish` and `fd` separatedly

**Reply by PatrickF1** [reply]
Thanks for reporting back here @Cokile. I'll add this to the Cookbook!

**Reply by kidonng** [reply]
If you are using `LS_COLORS` you should definitely checkout [vivid](https://github.com/sharkdp/vivid).
